src/components/Button/Button.scss

```diff
@@ -0,0 +1,33 @@
+.button {
+    justify-self: center;
+    border-radius: 5px;
+    border-width: 0px;
+    width: 40px;
+    height: 40px;  
+    font-weight: bold;
+    background-repeat: no-repeat;
+    background-position: center;
+    color: #EC823E;
+    background-color: #F7F8FD;
+
+    &--orange{
+      color: #FFE3BB;
+      background-color:#EC823E;
+    }
+        
+    &--plus{
+      background-image: url('../../images/icon-plus.svg');
+
+    }
+  
+    &--minus {
+      background-image: url('../../images/icon-minus.svg');
+
+    }
+  
+    &--add { 
+      width: 170px;

```

Comment:
The `width` of the button is better depending on the `padding-left` and `padding-right` and we rarely set a fixed width 

```
padding-left | text | padding-right
[     8px    ] Text [     8px     ]
```

Reply:
My suggestion is to get the `padding-left` and `padding-right` of the button and remove the width.